sessionctx/variable/varsutil.go

```diff
@@ -340,6 +340,15 @@ func ValidateSetSystemVar(vars *SessionVars, name string, value string) (string,
 			return "0", nil
 		}
 		return value, ErrWrongValueForVar.GenWithStackByArgs(name, value)
+	case LogBin:
```

Comment:
Could we put it to line 335? Then we can remove line344-351.

Reply:
LogBin need to return "ON" or "OFF", not "0" or "1"

Reply:
so i will remove  LogBin to line360

Reply:
so i will merge  'LogBin' and 'TiDBEnableTablePartition'

Reply:
@zimulala so i will merge 'LogBin' and 'TiDBEnableTablePartition'

Reply:
No. I mean you can merge `LogBin` with `GeneralLog`. `TiDBEnableTablePartition` has the value of `AUTO`, which is different from `LogBin`.

Reply:
,the return value is "0" or “1”, is it expected?

Reply:
The return value is "0" or “1”,  is it expected? @zimulala

Reply:
I think it's OK. We can handle `LogBin` like `SQLLogBin`.  Then we can remove `BoolToStatusStr`. 
We will handle #9357 in the next PR.

Reply:
ok, i will change the code later. @zimulala